tap_github/repository_streams.py

```diff
@@ -1407,3 +1919,125 @@ def parse_response(self, response: requests.Response) -> Iterable[dict]:
         th.Property("type", th.StringType),
         th.Property("site_admin", th.BooleanType),
     ).to_dict()
+
+
+class ProjectsStream(GitHubRestStream):
+    name = "projects"
+    path = "/repos/{org}/{repo}/projects"
+    # TODO I think this can be true, but I'm not sure.
```

Comment:
It's probably True most of the time. Very few children updates actually change the "updated_at" field of the parent (repository)